### DISCUSSION TITLE
[Dashboard] Refine Dashboard Capabilities: `details` could not clearly show all parameters and their type

### DISCUSSION THREAD

**Discussion by zzxwill** [kubevela/kubevela#823] [General] [answered]
- `details` could not clearly show all parameters and their type, how about justing keeping `reference`?
![image](https://user-images.githubusercontent.com/2805315/103117773-f8f12200-46a6-11eb-8c82-a04ffb5db4ce.png)

**Comment by zzxwill** [comment]
@sunny0826 Could you please help update the status of this issue?

**Comment by sunny0826** [comment, marked as answer]
I will optimise this feature.
